dlang.org.ddoc

```diff
@@ -54,9 +54,9 @@ DDOC=
 <!DOCTYPE html>
 <html lang="en-US">
 <!--
-    Copyright (c) 1999-$(YEAR) by Digital Mars
-    All Rights Reserved Written by Walter Bright
-    http://digitalmars.com
+    Copyright (c) 1999-$(YEAR) by the D Language Foundation
+    All Rights Reserved.
+    http://dlang.org/foundation.html
```

Comment:
I'm guessing @WalterBright is OK with what essentially indicates a copyright assignment and transfer of ownership to the D foundation :)